src/ports/postgres/modules/deep_learning/madlib_keras_predict.py_in

```diff
@@ -140,20 +140,23 @@ class Predict(BasePredict):
     def __init__(self, schema_madlib, model_table,
                  test_table, id_col, independent_varname,
                  output_table, pred_type, gpus_per_host,
-                 **kwargs):
+                 mst_key, **kwargs):
 
         self.module_name = 'madlib_keras_predict'
         self.model_table = model_table
+        self.mst_key = mst_key
```

Comment:
why don't we validate `mst_key` the same way we do for evaluate ?

Reply:
We don't validate the `mst_key` directly in any case. I'll add a `is_mult_model = mst_key is not None` line to make it uniform.